kickstarter/android-oss
# MBL-905: Remove Use of IA tab feature flag

# 📲 What

- Removed feature flag from source code
- Small refinement, new projects always have `AiDisclosure` model with ID available, but the rest of the fields are empty if not UI usage on the project, as before we were considering a project did not used AI if `AiDisclosure` was null

# 🤔 Why

- Clean up!
|  |  |

# 📋 QA

- Go to some project without use of AI, make sure the tab is not visible, like this one https://www.kickstarter.com/projects/babayagapatches/make-100-baba-yaga-hut-patches
- Go to a project with use of AI like this one https://www.kickstarter.com/projects/1099536463/avocat-ai-legal-assistant

# Story 📖

[https://kickstarter.atlassian.net/browse/MBL-905](MBL-905)
